GoogleForCreators/web-stories-wp
# Animation Panel: Fix pan regression and add translation safety

## Context
Part of fixit week. Pascal noticed some untranslated stuff in the default animation props, so the ticket was to look into that.

## Summary
While looking into the animation props it seems none of the untranslated strings were user facing so nothing really needed to be updated. Added some translations to labels in there just for extra safety, but none of them were user facing to begin with so 🤷 

While I was in here tho, I noticed all the pan animation directions were disabled for bg elements. It seems elements with type `media` have been updated to be of type `video` , `image` , or `gif` which caused a regression in the offset testing. I just updated it to accept the new more refined media types and now it works as it did before.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Pan direction for bg element animations should now work again.


## Testing Instructions
Go to story editor. Make an image a bg element on one page, a gif a bg element on another & a video bg element on another. See that the pan directions are properly enabling and disabling for the bg animation options on each type of bg element.


### QA
Follow testing instructions described above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7123
